apps/website/src/examples/Alert.informational.tsx

```diff
@@ -7,14 +7,15 @@ import { Alert } from '@itwin/itwinui-react';
 
 export default () => {
   return (
-    <Alert
-      type='informational'
-      onClose={() => console.log('CLOSED')}
-      clickableText='Learn more'
-      clickableTextProps={{ onClick: () => console.log('Clicked more info!') }}
-      style={{ minWidth: 'min(100%, 350px)' }}
-    >
-      This is an informational alert
+    <Alert type='informational' style={{ minWidth: 'min(100%, 350px)' }}>
+      <Alert.Icon type='informational' />
```

Comment:
I was getting the following error when trying to build the website

![image](https://user-images.githubusercontent.com/78564221/236085294-bf5de0d7-65b0-4bb9-bd23-fd24b5b86001.png)

Not sure why this error is only affecting the website and not storybook, so any help on this issue would be appreciated

Reply:
it's because you're using this
```ts
export type AlertIconProps<T extends React.ElementType = 'svg'> =
  PolymorphicComponentProps<T, AlertIconOwnProps>
```
instead of
```ts
export type AlertIconProps = PolymorphicComponentProps<'svg', AlertIconOwnProps>
```

Can you please change the whole file to use the correct format everywhere?

Reply:
This has been changed